demo/owner/owner.env

```diff
@@ -14,4 +14,8 @@ owner_to0_rv_blob=http://localhost:8042?ipaddress=127.0.0.1
 owner_svi_values=./serviceinfo/sample-values
 owner_svi_string=./serviceinfo/sample-svi.csv
 owner_api_user=apiUser
-owner_api_password=OwnerApiPass123
\ No newline at end of file
+owner_api_password=OwnerApiPass123
```

Comment:
The OnDie cert cache is provided for protocol samples only. We are running smoketest using demo folder which uses component-sample binaries. The component sample owner reads the ondie cache path from .env file. There is no default value provided in the .env file, so owner docker is running into "OnDie Initialization Error"

Also autoupdate property should be set to true by default because we are running component-samples via docker and the .cer and .crl files needs to be downloaded everytime the docker container is created.

Reply:
Changing initialization to support case where no OnDie configuration is specified in cases of backwards compatibility and when the user does not need to support OnDie. If and when we want to support OnDie for demo we can add them.

Reply:
done